elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -101,6 +101,7 @@ def process(self, pipeline):
 
         # unpack Cloud Object Storage configs
         cos_endpoint = runtime_configuration.metadata["cos_endpoint"]
+        cos_public_endpoint = runtime_configuration.metadata.get("cos_public_endpoint", cos_endpoint)
```

Comment:
This needs to reflect the [changed] property name in the schema

```suggestion
        cos_public_endpoint = runtime_configuration.metadata.get("public_cos_endpoint", cos_endpoint)
```

Reply:
ups